src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const BASE_PRICE = days * 40;
+  const MIDL_TERM = 3;
+  const LONG_TERM = 7;
+  const LOW_DISCOUNT = 20;
+  const HI_DISCOUNT = 50;
+
+  if (days < MIDL_TERM) {
+    return BASE_PRICE;
+  }
+
+  if (days >= MIDL_TERM && days < LONG_TERM) {
```

Comment:
The condition 'days >= MIDL_TERM' is redundant because it's already checked in the previous if condition.